### PR TITLE
Fake headset - controller button event

### PR DESCRIPTION
**Pull request by bcampbell**
Forgot to send events for the fake controller button.
Also moved the offset for the fake controller, so it's not situated at your feet.

### CODE REVIEW

File: src/headset/fake.c [hunk 1]
```diff
@@ -336,7 +366,14 @@ static ControllerHand fakeControllerGetHand(Controller* controller) {
 
 static void fakeControllerGetPosition(Controller* controller, float* x, float* y, float* z) {
   // for now, locked to headset
-  fakeGetPosition(x,y,z);
+  
+  float offset[3];
+  vec3_set(offset, 0, 0,-1.0f);
```
Comment (bjornbytes) [inline src/headset/fake.c]: Does this put the controller 1 meter in front of you?

Comment (bcampbell) [reply src/headset/fake.c]: yes, it does. I figured that made for more-or-less arms length, depending on your arms :- )

Comment (bjornbytes) [reply src/headset/fake.c]: Okay!  Just checking.  I'm a little weary at how rigid some of the fake headset stuff is, but I guess that was one of the tradeoffs of doing it as opposed to the raw keyboard/mouse APIs...

Comment (bcampbell) [reply src/headset/fake.c]: Yes. it'd be nice to be able to configure some of this stuff in `conf.lua`, but I'm wary of adding too much noise.
(eg I'd like to be able to set up two fake controllers - one offset to the left and one offset to the right, but it seems like a pretty niche requirement)

**Pull request merged**